pkgs/top-level/all-packages.nix

```diff
@@ -16875,6 +16875,8 @@ with pkgs;
 
   hydra-check = with python3.pkgs; toPythonApplication hydra-check;
 
+  ory-hydra = callPackage ../applications/misc/ory-hydra { };
```

Comment:
I'd place it near packages that start with `o`, although it has `hydra` in it's name.